src/ccl_neutrinos.c

```diff
@@ -10,20 +10,39 @@
 #include <gsl/gsl_roots.h>
 
 #include "ccl.h"
-#include "ccl_params.h"
-
 
 // Global variable to hold the neutrino phase-space spline
 gsl_spline* nu_spline=NULL;
```

Comment:
Should this be moved to the cosmology object with the other splines?

Reply:
Good question! There are few issues at play here. 

1. Right now we have to create objects in the following order. First you make a parameters structure and then you make a cosmology structure. At the time the spline above is built, the cosmology structure is not yet made. Thus putting this spline in it is a huge pain right now. We could do a larger refactor of the API here, but I would want a separate PR for that.

2. As far as I know, this spline doesn't actually need to be remade per cosmology. It holds an integral that is evaluated at different points for different cosmologies IIUIC. 

Thus I have left this code in place.

Reply:
Sounds good to me and we can revisit at a later PR